app/github/views.py

```diff
@@ -67,16 +68,17 @@ def github_callback(request):
                 'github_access_token': access_token
             })
 
-        # Update the user's session with handle and email info.
+        # Update the user's session with handle, email info, preffered language.
         session_data = {
             'handle': user_profile.handle,
             'email': user_profile.email,
             'access_token': user_profile.github_access_token,
             'profile_id': user_profile.pk,
             'name': user_profile.data.get('name', None),
             'access_token_last_validated': timezone.now().isoformat(),
-        }
-        for k, v in session_data.items():
+            LANGUAGE_SESSION_KEY: user_profile.pref_lang_code
```

Comment:
If the user doesn't have `pref_lang_code` set, we shouldn't manipulate `LANGUAGE_SESSION_KEY`.